src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,128 @@
+        positive-icon-color     blue-60-50
+        account-name-color      (if dark?
+                                  colors/white
+                                  colors/neutral-100)
+        percentage-change-color blue-60-50
+        currency-change-color   blue-60-50
+        dot-separator-color     (if dark?
+                                  (colors/custom-color :blue 50 40)
+                                  (colors/custom-color :blue 60 40))
+        loading-state-color     (if dark?
+                                  colors/neutral-90
+                                  colors/neutral-5)]
+    [rn/view
+     {:style style/account-overview-wrapper}
+     (if loading?
+       [loading-state loading-state-color]
+       [:<>
+        [rn/view {:style {:flex-direction :row}}
+         [quo2/text
+          {:weight :medium
+           :size   :paragraph-1
+           :style  (style/account-name account-name-color)}
+          account-name]
+         (when (= account :watched-address)
+           [icons/icon :i/reveal
+            {:container-style style/reveal-icon
+             :color           reveal-icon-color}])]
+        [quo2/text
+         {:weight :semi-bold
+          :size   :heading-1
+          :style  style/current-value}
+         current-value]
+        [rn/view
+         {:style style/row-centered}
+         [:<>
+          [quo2/text
+           {:weight :medium
+            :size   :paragraph-2
+            :style  (style/bottom-time-text (and (not= :custom time-frame)
+                                                 (seq time-frame-to-string)))}
+           (when (seq time-frame-string)
+             time-frame-string)]
+          (when (and (= :custom time-frame)
+                     (seq time-frame-to-string))
+            [:<>
+             [icons/icon :i/positive-right
+              {:size            16
+               :container-style style/right-arrow}]
+             [quo2/text
+              {:weight :medium
+               :size   :paragraph-2
+               :style  style/bottom-time-to-text}
+              time-frame-to-string]])]
+         (when (and (seq percentage-change)
+                    (seq currency-change))
+           [:<>
+            [quo2/text
+             {:weight :medium
+              :size   :paragraph-2
+              :style  (style/percentage-change percentage-change-color)}
+             percentage-change]
+            [rn/view
+             {:style (style/dot-separator dot-separator-color)}]
+            [quo2/text
+             {:weight :medium
+              :size   :paragraph-2
+              :style  (style/currency-change currency-change-color)}
+             currency-change]
+            [icons/icon
+             (if up?
+               :i/positive
+               :i/negative)
+             {:color positive-icon-color
+              :size  16}]])]])]))
+
+(def account-overview
+  "Create a account-overview UI component.
```

Comment:
Better to include on the internal where the api is

Reply:
But we want it to be available directly at the consumer's side.

It'll show on the IDE when hovering over it.

Reply:
Ah okay, sounds good 👌